spec/components/schemas.yaml

```diff
@@ -818,6 +818,150 @@ components:
       description: Feature identifier (unique per collection)
       example: smr200-200-4-2019
       type: string
+    itemGeometry:
+      oneOf:
+        - title: GeoJSON Point
+          type: object
+          required:
+            - type
+            - coordinates
+          properties:
+            type:
+              type: string
+              enum:
+                - Point
+            coordinates:
+              type: array
+              minItems: 2
+              items:
+                type: number
+            bbox:
+              type: array
+              minItems: 4
+              items:
+                type: number
+        - title: GeoJSON LineString
+          type: object
+          required:
+            - type
+            - coordinates
+          properties:
+            type:
+              type: string
+              enum:
+                - LineString
+            coordinates:
+              type: array
+              minItems: 2
+              items:
+                type: array
+                minItems: 2
+                items:
+                  type: number
```

Comment:
Might be a good idea to add a short description for the coordinates, for example also add the type of coordinate we support like in the bbox description.